### PR TITLE
Add Calculator Logic 🚀

### PR DESCRIPTION
**Pull request by hsztan**
I used the calculator 🧮 logic files and incorporated them with my GUI 🖥  calculator module. I also had to make 🛠 a helper method to deal with the UI display and create its respective state 🟢  to give it a more natural user 👤 experience look on the calculator screen. 🤓

### CODE REVIEW

File: src/components/calculator/Calculator.jsx [hunk 1]
```diff
@@ -1,80 +1,189 @@
...
+              className="calculator__keypad-button"
+            >
               AC
             </button>
-            <button type="button" className="calculator__keypad-button">
+            <button
+              onClick={this.handleClick}
+              type="button"
+              className="calculator__keypad-button"
+            >
```
Comment (Gambit142) [inline src/components/calculator/Calculator.jsx]: - [x] Kindly create another component as a child of the `Calculator` component and pass the component data as a props to the child component. You could put all the buttons in an array, map through the array and pass each button as a prop to the component. You can read this article to have a better understanding https://www.geeksforgeeks.org/how-to-map-data-into-components-using-reactjs/

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @hsztan 👍, 

Good job so far!

### Highlights

- Use of state ✔️
- Great use of logic files ✔️

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can also consider making the buttons readable. Currently, they are too tiny and that is not good accessibility practice because they are too tiny to be read. 👍  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/calculator/Calculator.jsx (hunk 1 above)
Comment (hsztan) [reply src/components/calculator/Calculator.jsx]: Understood. Will do 👍

Comment (hsztan) [reply src/components/calculator/Calculator.jsx]: Hi @Gambit142. Thanks for your feedback 💪. I was able to make a child component and render the buttons by sending its props and lifting the state to its parent 🤓

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @hsztan,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

### Some highlights for you
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator/Calculator.jsx (hunk 1 above)
Comment (Gambit142) [reply src/components/calculator/Calculator.jsx]: Hi @hsztan, Great work, you did it better than I suggested. 👍

**Pull request merged**